hack/jenkins/test-image/run.sh

```diff
@@ -42,6 +38,10 @@ export KUBE_COVERPROCS=8
 export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
 export LOG_LEVEL=4
 
+./hack/build-go.sh
+godep go install ./...
+./hack/install-etcd.sh
```

Comment:
maybe make this ./hack/install-etcd.sh || ./hack/travis/install-etcd.sh?

or we can cherrypick #15336 into release-1.0 and release-1.1.